projects/tad/index.md

```diff
@@ -0,0 +1,152 @@
+# Transparency of algorithmic decision making
```

Comment:
@uittenbroekrobbert shall we move this file to the docs directory? That way it will also appear on the site.

Reply:
Ah, that's where it should be, yes!

(On a sidenote: @robbertbos we may want to have a short description for each of our projects/workstreams there as well?)